pkg/scheduler/apis/config/types.go

```diff
@@ -168,8 +168,8 @@ type Plugins struct {
 	// Filter is a list of plugins that should be invoked when filtering out nodes that cannot run the Pod.
 	Filter *PluginSet
 
-	// PostFilter is a list of plugins that are invoked after filtering out infeasible nodes.
-	PostFilter *PluginSet
+	// PreScore is a list of plugins that are invoked before scoring.
+	PreScore *PluginSet
```

Comment:
This is an alpha API and is not used in current (e.g., v1.17) schedulers? Are config files in this format currently being read and acted upon?

Reply:
It's used in 1.17, and what we are planing is renaming it in 1.18.

Reply:
Config files using v1alpha1.postFilter is properly handled now, and it will be converted to preScore.

Reply:
@lavalamp Friendly ping, and do I need to do some other things for the api-review?
😄

Reply:
It seems like this will break existing files, or at least cause them to do something different than the author expected?

Reply:
Existing files are `v1alpha1`, which get a conversion for `postFilter` to `preScore`.

Reply:
This is actually tested in `options_test.go`

Reply:
Ah great I missed that this was v1alpha2. There's not v1alpha2 files out in the wild, only v1alpha1, right?

Reply:
Correct, v1alpha2 is what we are introducing in this release :)